MyString.java

```diff
@@ -64,8 +92,15 @@ public static String spacedString(String str) {
      * @return a randomly generated string, consisting of 'n' lowercase letters
      */
     public static String randomStringOfLetters(int n) {
-        //// Replace the following statement with your code
-        return null;
+        String randStr = "";
+        for (int i = 0; i < n; i++) {
+            double randNum = Math.random();
+            int charNum = (int) (randNum * 26 );
+            char randChar = 97;

```

Comment:
26, 97 magic values, please use constant or formula for first, and 'a' for second